news/_posts/2021-02-24-meet-us-second-oss-demo.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Second public demo from the maintainers team"
+subtitle: "Meet us at the second session of the open source project's public demo"
+date:   2021-02-24 13:00:00
+authors: [ MathieuFerment, PabloBorowicz ]
```

Comment:
```suggestion
authors: [ MathieuFerment ]
```

This one you wrote all by yourself. Alternatively, you can also sign it as the prestashop team instead